python/src/trezorlib/cardano.py

```diff
@@ -20,19 +20,139 @@
 from .tools import expect
 
 PROTOCOL_MAGICS = {"mainnet": 764824073, "testnet": 42}
+NETWORK_IDS = {"mainnet": 1, "testnet": 0}
 
 REQUIRED_FIELDS_TRANSACTION = ("inputs", "outputs")
 REQUIRED_FIELDS_INPUT = ("path", "prev_hash", "prev_index")
 
+INCOMPLETE_OUTPUT_ERROR_MESSAGE = "The output is missing some fields"
+
+
+def create_address_parameters(
+    address_type: messages.CardanoAddressType,
+    spending_key_path_str: str,
```

Comment:
please rename these to match protobuf (see comment there)

also this function should accept parsed paths instead of calling `parse_path` here

Reply:
Done [here](https://github.com/trezor/trezor-firmware/pull/1112/commits/aa4c7374f5a2985426319d47fe36b58392e7e6f9).